kale/embed/mogonet.py

```diff
@@ -0,0 +1,121 @@
+# =============================================================================
+# Author: Sina Tabakhi, stabakhi1@sheffield.ac.uk
+# =============================================================================
+
+"""
+Construct a message passing network using PyTorch Geometric for MOGONET method.
```

Comment:
API has a higher requirement on documentation. Explain what is Mogonet, most do not know.

Reply:
Your doc also does not sound an embed module. Tell people what in and what out.